admin-manual/howto.rst

```diff
@@ -289,6 +287,26 @@ Per la riattivazione:
   signal-event nethcti-server3-update
 
 
+Attivazione della modalità Click2Call automatico su Cloud
+---------------------------------------------------------
+
+Nel caso in cui il centralino telefonico fosse in cloud
+e i telefoni siano in LAN dietro NAT, per poter utilizzare la modalità
+Click2Call automatico è necessario eseguire i comandi:
```

Comment:
Nel caso in cui il centralino telefonico fosse in una rete differente dai telefoni e comunichi con loro attraversando dei servizi di NAT, ad esempio centralino in cloud e telefoni in LAN dietro NAT, mentre il pc con il quale si accede al |product_cti| comunica direttamente con il telefono, ad esempio si trova nella stessa LAN dei telefoni, per poter utilizzare la modalità Click2Call automatico è necessario eseguire i comandi:

Reply:
done